golang/1.9/Dockerfile

```diff
@@ -8,7 +8,10 @@ RUN apk --update add build-base git bash \
     && go get -v -u github.com/stretchr/testify \
                 github.com/tebeka/go2xunit \
                 github.com/t-yuki/gocover-cobertura \
-                gopkg.in/alecthomas/gometalinter.v2 \
-    && gometalinter.v2 --install
+                gopkg.in/alecthomas/gometalinter.v1 \
```

Comment:
Why is it changed to v1?

Reply:
whoops psps copy-pasta error. good catch.